api/v1alpha1/storageconsumer_types.go

```diff
@@ -46,8 +45,12 @@ type StorageConsumerSpec struct {
 type CephObjectsSpec struct {
 	// BlockPoolName holds the name of created ceph block pool.
 	BlockPoolName string `json:"blockPoolName,omitempty"`
-	// CephUser holds the name of created ceph user.
-	CephUser string `json:"cephUser,omitempty"`
+	// CephRbdUser holds the name of created ceph user required for rbd.
+	CephRbdUser string `json:"cephRbdUser,omitempty"`
```

Comment:
RBD is an acronym so it should be `CephRBDUser`, or else we can use `CephBlockPoolUser`.